.circleci/config.yml

```diff
@@ -112,6 +117,14 @@ workflows:
       - "another-locale":
           {}
 
+      - "windows-server-2022":
+          name: "Windows Server 2022, CPython <<matrix.pythonVersion>>"
+          matrix:
+            parameters:
+              pythonVersion:
+                - "3.9"
+                - "3.11"
```

Comment:
On GitHub Actions we tested 4 versions of CPython but:

* the CircleCI Windows image only has these two CPythons installed so testing other versions is more expensive (development and runtime)
* Tahoe's an application anyway, maybe we don't need to support a huge range of Python versions
* Python 3.8 and 3.10 are still covered by other CI configurations (just not Windows configurations)

Reply:
:+1:
(might be worth putting at least some of that in a comment ... "just in case" CircleCI changes what pythons they support ;) e.g. "these were chosen because they're versions that CircleCI supports for Windows")